src/GraphQL.NewtonsoftJson/GraphQLContractResolver.cs

```diff
@@ -21,6 +23,24 @@ protected override JsonConverter ResolveContractConverter(Type objectType)
             if (typeof(ExecutionResult).IsAssignableFrom(objectType))
                 return new ExecutionResultJsonConverter(_errorInfoProvider, NamingStrategy);
 
+            if (typeof(Inputs) == objectType)
+                return new InputsJsonConverter();
+
+            if (typeof(GraphQLRequest) == objectType)
+                return new GraphQLRequestJsonConverter();
+
+            if (objectType == typeof(IEnumerable<GraphQLRequest>) ||
```

Comment:
You may add static internal method in GraphQLRequestListJsonConverter to check all types in one place and call this method from here.